assets/app/view/game/round/stock.rb

```diff
@@ -57,6 +57,12 @@ def render
 
           children.concat(render_buttons)
           children.concat(render_failed_merge) if @current_actions.include?('failed_merge')
+          children << render_bidbox(@game.bidbox_minors) if @current_actions.include?('bidbox_minors') &&
+            @game.respond_to?(:bidbox_privates)
+          children << render_bidbox(@game.bidbox_concessions) if @current_actions.include?('bidbox_concessions') &&
+            @game.respond_to?(:bidbox_concessions)
+          children << render_bidbox(@game.bidbox_privates) if @current_actions.include?('bidbox_privates') &&
+            @game.respond_to?(:bidbox_privates)
```

Comment:
Why do these need to be actions? Isn't the action always bid?

Reply:
Sure i can remove the action, bidbox_.. and just use bid and still check the rspond_to?

Reply:
Either `respond_to?` or something like a `can_bid?(player, entity)` method that is always false in the base class.

Reply:
what is all of this?